desktop/ui/src/main/java/org/datacleaner/windows/DataCloudLogInWindow.java

```diff
@@ -0,0 +1,294 @@
+                        )
+        );
+
+        layout.setHorizontalGroup(layout.createParallelGroup()
+                .addComponent(informationText)
+                .addGroup(layout.createSequentialGroup()
+                        .addGap(0, 0, Integer.MAX_VALUE)
+                        .addGroup(layout.createParallelGroup()
+                                .addComponent(usernameLabel)
+                                .addComponent(passwordLabel)
+                        )
+                        .addGap(5)
+                        .addGroup(layout.createParallelGroup()
+                                .addComponent(usernameTextField, GroupLayout.DEFAULT_SIZE, GroupLayout.DEFAULT_SIZE, 250)
+                                .addComponent(passwordTextField, GroupLayout.DEFAULT_SIZE, GroupLayout.DEFAULT_SIZE, 250)
+                        )
+                        .addGap(0, 0, Integer.MAX_VALUE)
+                )
+                .addComponent(invalidCredentialsLabel)
+                .addComponent(dontShowAgainCheckBox)
+                .addGroup(layout.createSequentialGroup()
+                        .addGap(0, 0, Integer.MAX_VALUE)
+                        .addComponent(signInButton)
+                        .addGap(0, 0, Integer.MAX_VALUE)
+                )
+        );
+
+        // 3. Add listeners
+        // TODO: don't remember on click the checkbox, but on dialog close.
+        dontShowAgainCheckBox.addActionListener(new DisableShowDialog(dontShowAgainCheckBox));
+        signInButton.addActionListener(new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                signIn();
+            }
+        });
+        ClearErrorLabelDocumentListener clearErrorListener = new ClearErrorLabelDocumentListener();
+        usernameTextField.getDocument().addDocumentListener(clearErrorListener);
+        passwordTextField.getDocument().addDocumentListener(clearErrorListener);
+
+        return result;
+    }
+
+    class ClearErrorLabelDocumentListener implements DocumentListener {
+
+        @Override
+        public void insertUpdate(DocumentEvent e) {
+            invalidCredentialsLabel.setText("");
+        }
+
+        @Override
+        public void removeUpdate(DocumentEvent e) {
+            invalidCredentialsLabel.setText("");
+        }
+
+        @Override
+        public void changedUpdate(DocumentEvent e) {
+            invalidCredentialsLabel.setText("");
+        }
+    }
+
+    @Override
+    protected String getBannerTitle() {
+        return "Sign In / Register";
+    }
+
+    @Override
+    protected int getDialogWidth() {
+        return 500;
+    }
+
+    @Override
+    protected JComponent getDialogContent() {
+        return _contentPanel;
+    }
+
+
+    protected boolean isWindowResizable() {
+        return false;
+    }
+
+    @Override
+    public String getWindowTitle() {
+        return "DataCloud Sign In";
```

Comment:
Wording wise I think it makes more sense to formulate it like: "Sign in to DataCloud"